### PR TITLE
fix: keep space between deep selector `>>>`

### LINKED ISSUE #3790: Keep space between vue deep selector ">>>" ?
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeeBbADgGwIbwB8AOlAARmoAmAlgG5lj4DOzAvMSAEYQAenhVAHpadElGGZcBOONKpmYAE40sMcRTi8sEJTDJU4AMzwBXHPuCkKFKHgxwkZAOQAVOMxjPrZAL4BueSFFFTU5CU8ATxw4MkUILDgqDQohITIASTIAdxpmAAsyGHyCMgBrODgsOKw8MFiuOBhsyvI6U1jDKri4GLAYXTIBQhIQABoyI0Gte1xHHzI0sgA6Hl4yEcIVsHyaHCoAWkhsaAQYH1W+DZHt3f2jiBOoM7Irchsp2APmGgAvRzIACYACxYXiBd6+IJRGKEcYgBIwGjQZjIUB4JRKCDZAAKGIQqJQeDoEBoVHheE8yBMOGYcDG3CUdQqMAAyrUwDQoABzZAwJQdBlUCBgal4Wn0kBcul6HFM7kYPBiiUMgBWzF4ACEmWAWaz7HAADJcuDKukMiCmGBYK2As2S2pKGXIEDtU0MrAqWAAdTJxWQAA4AAwerF071MrAuz0eOBKOjukBKOAAR1MNGTcrwCqVSBp5pAdIwND5AslPx5MQAiqYIPBS4KQDA8FxfVR-UhAQz+Xg9lzuQBhR6Kl1QU7w0x01wtwn5uC+XxAA)
```sh
--parser vue
--single-quote
```

**Input:**
```vue
<template>
  <div class="box"></div>
</template>

<script>
  export default {
    name: 'Test'
  };
</script>

<style scoped>
   // I wish that keep space between vue deep selector ">>>", for example:
   // .box >>> .child-component
  .box >>> .child-component {
    font-size: 24px;
  }
</style>
```

**Output:**
```vue
<template>
  <div class="box"></div>
</template>

<script>
export default {
  name: 'Test'
};
</script>

<style scoped>
// I wish that keep space between vue deep selector ">>>", for example:
// .box >>> .child-component
.box>>>.child-component {
  font-size: 24px;
}
</style>

```

**Expected behavior:**
```vue
<style scoped>
.box >>> .child-component {
  font-size: 24px;
}
</style>

```

### PR DESCRIPTION
**Pull request by evilebottnawi**
Fixes #3790

### CODE REVIEW

File: tests/css_selector_list/selectors.css [hunk 1]
```diff
@@ -71,3 +71,19 @@
...
+  .child-two {}
+}
+.parent >>> {
+  .child-one {}
+  .child-two {}
+}
+.parent  >>>  {
+  .child-one {}
+  .child-two {}
+}
```
Comment (j-f1) [inline tests/css_selector_list/selectors.css]: Could you add a few similar tests where the `>>>` is before the `.child-x` selectors?

Comment (evilebottnawi) [reply tests/css_selector_list/selectors.css]: @j-f1 yep, but it is invalid css, should we handle invalid css?

Comment (lydell) [reply tests/css_selector_list/selectors.css]: Might be valid SCSS though

Comment (evilebottnawi) [reply tests/css_selector_list/selectors.css]: @lydell right now `scss` don't support deep selector, also for `scss` valid only `.parent [combinator] { .child { color: red }}` structure

**Comment by evilebottnawi** [comment]
@j-f1 @lydell done, seems we should update snapshots (ci broken, local also), but i think it is not related to this PR

**Comment by j-f1** [comment]
Can you run Prettier on your changed code? It looks like it wants you to break the conditional onto multiple lines.

**Comment by evilebottnawi** [comment]
@j-f1 oh, stupid error :smile:

**Comment by evilebottnawi** [comment]
Don't merge.
After merge this https://github.com/prettier/prettier/pull/3800 i can do rebase and add more tests

**Comment by evilebottnawi** [comment]
@j-f1 friendly ping :+1:

**Pull request merged**